src/Summoner/CLI.hs

```diff
@@ -31,27 +36,52 @@ summon = execParser prsr >>= runWithOptions
 
 -- | Run 'hs-init' with cli options
 runWithOptions :: InitOpts -> IO ()
-runWithOptions (InitOpts projectName targets) = do
-     -- Generate the project.
-    generateProject projectName targets
+runWithOptions (InitOpts projectName cliConfig maybeFile) = do
+    file <- case maybeFile of
+        Nothing -> (</> "summoner.toml") <$> getHomeDirectory
```

Comment:
Maybe we can put `"summoner.toml"` into `Default` module. As well as `IO` function which returns full path to this file.